### PR TITLE
Dev 1627 verify email

### PR DESCRIPTION
**Pull request by sierra-henderson**
[changelog]
added: DEV-1627 Partner can verify their email address

### CODE REVIEW

File: src/app/auth/services/app-entry.service.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+export class AppEntryService {
+  state: any
+
+  constructor(
+    private userSessionService: UserSessionService,
+    private router: Router,
+    private feedback: UserFeedbackService,
+  ) {}
+
+  getIn(): Observable<boolean> {
```
Comment (ShiftyMcCool) [inline src/app/auth/services/app-entry.service.ts:17]: I'm not seeing much use for this service in this app.  the getIn() method is much simpler that customer-portal and really doesn't add much here. Thoughts?

File: src/app/auth/sign-up/change-email-modal/change-email-modal.component.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<form
```
Comment (ShiftyMcCool) [inline src/app/auth/sign-up/change-email-modal/change-email-modal.component.html:1]: Doesn't look like this is a modal anymore, might wanna change the file/component name

**Review by ShiftyMcCool — APPROVED** [review]
Looks good, just a few minor tweaks and 👍

File: src/app/auth/services/app-entry.service.ts (hunk 1 above)
Comment (sierra-henderson) [reply src/app/auth/services/app-entry.service.ts:17]: If we aren't doing authorizations in this app, sure. For now, I'll delete it

**Pull request merged**